bin/install

```diff
@@ -6,87 +6,24 @@ set -eu -o pipefail
 source "$(dirname "$0")/../lib/utils.sh"
 
 install_nodejs() {
-  local install_type="$1" version_query="$2" install_path="$3"
-  local version=
-
-  version="$(resolve_version_query "$version_query")"
-
-  if [ "$version" != "$version_query" ]; then
-    install_aliased_version "$version" "$version_query" "$install_path"
-  else
-    install_canon_version "$install_type" "$version" "$install_path"
-  fi
-}
-
-
-try_to_update_nodebuild() {
-  local exit_code=0
-
-  "$ASDF_NODEJS_PLUGIN_DIR/lib/commands/command-update-nodebuild.bash" 2> /dev/null || exit_code=$?
-
-  if [ "$exit_code" != 0 ]; then
-    printf "
-$(colored $YELLOW WARNING): Updating node-build failed with exit code %s. The installation will
-try to continue with already installed local defintions. To debug what went
-wrong, try to manually update node-build by running: \`asdf %s update nodebuild\`
-\n" "$exit_code" "$ASDF_NODEJS_PLUGIN_NAME"
-  fi
-}
-
-
-install_canon_version() {
   local install_type="$1" version="$2" install_path="$3"
   local args=()
 
+  version=$(resolve_version "$version")
+
   if [ "$install_type" = ref ] || [ "${ASDF_NODEJS_FORCE_COMPILE-}" ]; then
     args+=(-c)
   fi
 
-  try_to_update_nodebuild
+  if ! [ "${ASDF_NODEJS_SKIP_NODEBUILD_UPDATE-}" ]; then
+    try_to_update_nodebuild
+  fi
```

Comment:
nitpick: This `ASDF_NODEJS_SKIP_NODEBUILD_UPDATE` could be moved inside the function, as this block of code is duplicated in two places.

For example, instead of this it'd be:

```bash
maybe_try_to_update_nodebuild

...
maybe_try_to_update_nodebuild() {
  if ! [ "${ASDF_NODEJS_SKIP_NODEBUILD_UPDATE-}" ]; then
    # Update code here...
  fi
fi
```